docs/source/tutorials/linops/linear_operators_quickstart.ipynb

```diff
@@ -0,0 +1,622 @@
+{
```

Comment:
<p>Maybe make more clear the memory requirement benefit:</p><p> </p><p>"This is an implicit definition that does not require the storage of the $n^2$ entries of the matrix form of $A$...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/534/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>changed</p>